src/iocore/cache/StripeSM.h

```diff
@@ -65,6 +65,8 @@ class StripeSM : public Continuation, public Stripe
 {
 public:
   CryptoHash hash_id;
+  char      *path = nullptr;
+  int        fd{-1};
```

Comment:
`CacheDisk` has `path` and `fd`. I start wondering we should just refer them instead of copy it because `StripeSM` has `CacheDisk *`.
https://github.com/apache/trafficserver/blob/d0b6a4b49ba8f2738fe91606c9cea55b06220cec/src/iocore/cache/P_CacheDisk.h#L86
https://github.com/apache/trafficserver/blob/d0b6a4b49ba8f2738fe91606c9cea55b06220cec/src/iocore/cache/Cache.cc#L291

Reply:
Sure enough. We should definitely deduplicate that! Good find.

Reply:
Changes made in #11895 and #11896.